site/source/user-manual/connecting/connecting-tor/tor-firefox/torff-android.rst

```diff
@@ -7,45 +7,50 @@ Configuring Firefox for Tor on Android
 .. caution::
   This guide assumes you have completed :ref:`setting up Tor for Android<tor-android>`. Please visit this section before proceeding as it is required for Firefox to properly work with Tor.
 
+.. note:: Setups may strongly vary across different Android forks. We have made specific mention of GrapheneOS which has been tested on Pixel 5 and Pixel 6.
```

Comment:
Do we know that this will not work on CalyX?  They share the same codebase I think - and if this works on Lineage they should all be mentioned

Reply:
I don't know. I'm loathe to mention them without actually testing them.

Reply:
Let's chat in the AM

Reply:
so do we mention that it's tested and doesn't work or just not mention? @kn0wmad

Reply:
I think leave your note about the range of android flavors and then just make note on the step that is for graphene/calyx rather than what platforms are tested.  We can add here if necessary

Reply:
can we actually test on calyx first before we do that?

Reply:
I don't have a calyx device, maybe Dread does?